.github/workflows/ci.yml

```
@@ -217,43 +217,76 @@ jobs:
         name: dist
         path: wheelhouse/*
 
-  build-binary:
-    name: Binary wheels
+  build-macos:
+    name: MacOS
     strategy:
       matrix:
         pyver: [3.6, 3.7, 3.8, 3.9]
-        os: [macos, windows]
       fail-fast: false
-    runs-on: ${{ matrix.os }}-latest
+    runs-on: macos-latest
     needs: pre-deploy
     steps:
     - name: Checkout
       uses: actions/checkout@v2
       with:
         submodules: true
-    - name: Setup Python 3.8
+    - name: Setup Python
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.pyver }}
     - name: Cythonize
-      if: ${{ matrix.no-extensions == '' }}
       run: |
         make cythonize
     - name: Install dependencies
       run: |
         python -m pip install -U setuptools wheel
     - name: Make wheel
-      run:
+      run: |
+        python setup.py bdist_wheel
```

It's highly discouraged to invoke setuptools directly.
```suggestion
        python -m pip wheel -w dist/
```

---

(in fact, the today-recommended method is to use [python-build](https://github.com/pypa/build) front-end for building both source and built dists)